src/yoti_common/constants.js

```diff
@@ -23,4 +23,10 @@ module.exports = Object.freeze({
   ATTR_POSTAL_ADDRESS: 'postal_address',
   ATTR_DOCUMENT_DETAILS: 'document_details',
   ATTR_STRUCTURED_POSTAL_ADDRESS: 'structured_postal_address',
+  CONTENT_TYPE_UNDEFINED: 0,
```

Comment:
If these are only going to be used in `attribute.converter.js`, do you think we should keep them in that file? And we can move them to this file if they're going to be used in other places

Reply:
I've now moved these